browser/resources/brave_new_tab_page_refresh/stories/sb_locale.ts

```diff
@@ -96,13 +160,18 @@ const localeStrings: { [K in StringKey]: string }  = {
   widgetSettingsTitle: 'Widgets'
 }
 
-export function createLocale() {
-  const getString =
-    (key: string) => String((localeStrings as any)[key] || 'MISSING')
+const pluralStrings: { [K in PluralStringKey]: string }  = {
+  newsSourceCountText: '# sources'
+}
+
+export function createLocale(): Locale {
   return {
-    getString,
-    async getPluralString (key: string, count: number) {
-      return getString(key).replace('#', String(count))
+    getString(key) {
+      return String((localeStrings as any)[key] || 'MISSING')
```

Comment:
Hey we shouldn't need the `as any` or `String()` now

```suggestion
      return localeStrings[key] ?? 'MISSING'
```

Reply:
https://www.typescriptlang.org/play/#code/C4TwDgpgBA0hIGcoF4oCID2A7CaoB91gB3DPQtYACwCcJcAoBgY2wWCgDMAuKAJQisaAEwA8cRABoo7GgEssAcwB8KKAG8GUbemy5eaAPI40krTsqk0BgCpWzOorXrX0N54wC+TTgFcszMBy2FCKEMAAysDySgAUANbwvBIIAJQa5tp0wL40WFwA2okgALpQAPzl6ACyAJIREbUAcgDiaAyeQA